src/Core/src/Credentials/IniFileLoader.php

```diff
@@ -62,6 +62,7 @@ public function loadProfiles(array $filepaths): array
 
             foreach ($this->parseIniFile($filepath) as $name => $profile) {
                 $name = preg_replace('/^profile /', '', (string) $name);
+                \assert($name !== null);
```

Comment:
how is it possible to get null here ?

Reply:
Note that phpstan would also complain about those at level 7 (for cases returning `false`) or 8 (for cases using `null`). But our current config runs at level 6 only.